app/src/Containers/EventSearchContainer.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { ScrollView, SafeAreaView } from 'react-native'
+import TopOfApp from '@/Components/TopOfApp'
+import underDevelopmentAlert from '@/Utils/UnderDevelopmentAlert'
+import DateTimePicker from '@react-native-community/datetimepicker'
+import FreeSearchBar from '@/Components/FreeSearchBar'
+
+const Heading = styled.Text`
+  font-weight: bold;
+  font-size: 18px;
+  margin: 15px 15px 0px 15px;
+`
+const Label = styled.Text`
+  font-weight: bold;
+  font-size: 14px;
+  margin: 15px 15px 0px 15px;
+`
+const SectionView = styled.View`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+`
+const EventButton = styled.TouchableOpacity`
```

Comment:
As this is the same button here and in the projects `SearchContainer`, could this be pulled out as a general component in its own file (e.g. `QuickSearchButton`) that could be used by both containers?  Be great if we can make components reusable where possible (and makes life easier if we ever need to update them).

Reply:
Good point, well made, done!